forc/GFS/README.md

```diff
@@ -0,0 +1,67 @@
+## Description
+
+* This package generates DATM input data for the forcing fluxes at the sea surface from the NCEP operational GFSv16 outputs.
+
+## Inputs of conversion in conv_gfs2datm_long_beta4.f
+
+* INPUT variables : from the output of GFSv16,
+    ==> longitude, latitude, time, DLWRF_surface, ULWRF_surface, DSWRF_surface, UFLX_surface, VFLX_surface, SHTFL_surface,
+      LHTFL_surface, PRATE_surface, UGRD_10maboveground, VGRD_10maboveground, PRES_surface, TMP_1hybridlevel, 
+      SPFH_1hybridlevel, UGRD_1hybridlevel, VGRD_1hybridlevel, SPFH_2maboveground, TMP_2maboveground, LAND_surface, 
+      delz, PRATE_surface, pres_hyblev1, precp, fprecp, ICEC_surface
```

Comment:
There are also a lot more variables being read in the fortran code e.g. `SOILW`, `SOILL`, etc. that seem to be redundant in creating the forcing file.

Reply:
In the process of code, it is needed to read by order for the input netcdf.

Reply:
why not remove them when using `wgrib2` to convert to netcdf as you do for `PRATE, UFLX and VFLX`.